tex/alg-NT/dedekind.tex

```diff
@@ -125,6 +125,17 @@ \section{Ideal arithmetic}
 So ``scaling'' and ``multiplying by principal ideals'' are the same thing.
 This is important, since we'll be using the two notions interchangeably.
 \end{remark}
+\begin{remark}
+	The addition of two ideals does not correspond to the addition of elements --- for example, $(4)+(6)=(4, 6)=(2)$, but $4+6=10$.
+
+	This is the best we can hope for --- addition of elements does not make sense for ideals --- for example, $1+1=2$ and $1+(-1)=0$, but as ideals, $(1)=(-1)$.
+
+	In fact, addition of ideal is the straightforward generalization of $\gcd$ of elements --- as you can check in the example above, $\gcd(4, 6)=2$.
+
+	Nevertheless, I hope you agree that $\ka+\kb$ is a natural notation, compared to something like $(\ka, \kb)$.
+
+	Because factorization involves \emph{multiplying}, instead of adding, the ideals together, we will not need to use the notation $\ka+\kb$ any time soon.
```

Comment:
(indeed, I can't find the notation being used anywhere in the following chapters)